docs/src/main/tut/getting-started.md

```diff
@@ -15,26 +15,34 @@ libraryDependencies += "com.cibo" %% "evilplot" % "0.2.0" // Use %%% instead of
 
 EvilPlot is all about building larger graphical capabilities out of smaller ones. What this means for you is that making
 simple plots is easy, and we don't have to spend a lot of time going over all the features just to get started. So let's make
-our first plot, a simple scatter plot with sequential x-values and random y-values.
+our first plot, a simple scatter plot with sequential x-values and random y-values. Then we'll save the plot as an image
+file to disk.
 
 <div class="row">
 <div class="col-md-6" markdown="1">
 ```scala
 import com.cibo.evilplot.plot._
 import com.cibo.evilplot.plot.aesthetics.DefaultTheme._
 import com.cibo.evilplot.numeric.Point
+import javax.imageio.ImageIO
+import java.io.File
 
 val data = Seq.tabulate(100) { i =>
   Point(i.toDouble, scala.util.Random.nextDouble())
 }
-ScatterPlot(data).render()
+val plotDrawable = ScatterPlot(data).render()
+val plotImage = plotDrawable.asBufferedImage
```

Comment:
There is a `write` method on `Drawable` in addition to `asBufferedImage`. It might be cleaner looking for explanatory purposes to just do `ScatterPlot(data).render().write(...) ` instead of separate vals for each thing.

Reply:
Ah, implicits. Nice, yea that looks much better. I had just grabbed this out of the demo plot writer